### PR TITLE
Ports some recent /tg/ medical changes.

### PR DESCRIPTION
**Pull request by lolman360**
## About The Pull Request

Adds Trophazole, Mitogen Factor, Peptides, Rhigoxane, Thizalid and Oxzalid from /traditionalgames/station. Thizalid and Oxzalid don't do liver damage, and Thizalid ODs at 10u instead of 6.

## Why It's Good For The Game

These chemicals are more interesting than the current bicard/kelo/antitox core ones, more powerful in their specific niches and harder to make. They make a good intermediate between bicard/kelo/antitox and penetic/saliacid/oxalandrone as they don't need a specific amount of damage to work at their full power, but instead need a specific delivery method.

(kev please no "CHEM WAREFARE BAD!!" comment)

## Changelog
:cl:
add: NT has updated the station's chemical infrastructure to allow production of Trophazole, Thizalid and Rhigoxane.
/:cl:

### CODE REVIEW

**Comment by Linzolle** [comment]
this PR title scared me for a moment

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -624,9 +624,9 @@
 	desc = "The Multi-Augmented Severe Operations Networked Resource Integration Gear is an man-portable tank designed for extreme environmental situations. It is excessively bulky, but rated for all but the most atomic of hazards. The specialized armor is surprisingly weak to conventional weaponry. The exo slot can attach most storge bags on to the suit."
 	icon_state = "hardsuit-ancient"
 	item_state = "anc_hardsuit"
-	armor = list("melee" = 10, "bullet" = 5, "laser" = 5, "energy" = 500, "bomb" = 500, "bio" = 500, "rad" = 500, "fire" = 500, "acid" = 500)
+	armor = list("melee" = 25, "bullet" = 25, "laser" = 25, "energy" = 100, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
```
Comment (Poojawa) [inline code/modules/clothing/spacesuits/hardsuit.dm]: undocumented change

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -624,9 +624,9 @@
...
 	slowdown = 6 //Slow
-	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/storage, /obj/item/construction/rcd, /obj/item/pipe_dispenser)
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/construction/rcd, /obj/item/pipe_dispenser)
```
Comment (Poojawa) [inline code/modules/clothing/spacesuits/hardsuit.dm]: undocumented change

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 3]
```diff
@@ -636,7 +636,7 @@
 	desc = "The M.A.S.O.N RIG helmet is complimentary to the rest of the armor. It features a very large, high powered flood lamp and robust flash protection."
 	icon_state = "hardsuit0-ancient"
 	item_state = "anc_helm"
-	armor = list("melee" = 10, "bullet" = 5, "laser" = 5, "energy" = 500, "bomb" = 500, "bio" = 500, "rad" = 500, "fire" = 500, "acid" = 500)
+	armor = list("melee" = 25, "bullet" = 25, "laser" = 25, "energy" = 100, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
```
Comment (Poojawa) [inline code/modules/clothing/spacesuits/hardsuit.dm]: undocumented change

**Review by Poojawa — CHANGES_REQUESTED** [review]
🤔

**Comment by YakumoChen** [comment]
>he put commits from other PRs in Master

**Comment by lolman360** [comment]
>mfw forgot to delete and reopen fork to have a clean fork
>mfw now have to get gitdesktop to revert commits since webgit dumb

**Comment by CameronWoof** [comment]
Trophazole won't do anything useful, as you didn't port the reaction that causes metafactor to turn into peptides.

**Comment by lolman360** [comment]
not yet i had to go out

**Comment by lolman360** [comment]
done

**Comment by lolman360** [comment]
travis is shitting itself because pr is half modularized, on it

**Pull request closed without merging**